src/app/utils/persistentStorageTemplate.ts

```diff
@@ -0,0 +1,54 @@
+export const fileStore = `"file-store": {
+  "shared": false,
+  "data": {
+    "path": "data"
+  },
+  "index": {
+    "path": "index"
+  },
+  "write-behind": {
+    "modification-queue-size": "2048"
+  }
+}`;
+
+export const remoteStore = `"remote-store": {
+  "cache": "mycache",
+  "raw-values": "true",
```

Comment:
The default value is "false" and is applicable for most use-cases, so we should remove this.